### PR TITLE
feat(code-runner): added the toggletasks.nvim plugin

### PR DESCRIPTION
**Pull request by vbmaarten**
## 📑 Description

added the toggletasks.nvim plugin

### CODE REVIEW

File: lua/astrocommunity/code-runner/toggletasks-nvim/init.lua [hunk 1]
```diff
@@ -0,0 +1,15 @@
+return {
+  "jedrzejboczar/toggletasks.nvim",
+  dependencies = {
+    { "nvim-lua/plenary.nvim" },
+    { "akinsho/toggleterm.nvim" },
+    { "nvim-telescope/telescope.nvim" },
+  },
+  opts = {},
+  init = function() require("telescope").load_extension "toggletasks" end,
```
Comment (mehalter) [inline lua/astrocommunity/code-runner/toggletasks-nvim/init.lua:9]:
```suggestion
  init = function()
    require("astrocore").on_load("telescope.nvim", function() require("telescope").load_extension "toggletasks" end)
  end,
```

respect the lazy loading of telescope

File: lua/astrocommunity/code-runner/toggletasks-nvim/init.lua [hunk 2]
```diff
@@ -0,0 +1,15 @@
+return {
+  "jedrzejboczar/toggletasks.nvim",
+  dependencies = {
+    { "nvim-lua/plenary.nvim" },
+    { "akinsho/toggleterm.nvim" },
+    { "nvim-telescope/telescope.nvim" },
```
Comment (mehalter) [inline lua/astrocommunity/code-runner/toggletasks-nvim/init.lua:6]:
```suggestion
    { "nvim-telescope/telescope.nvim" },
    {
      "AstroNvim/astrocore",
      opts = {
        mappings = {
          n = {
            ["<Leader>tss"] = { "<Cmd>Telescope toggletasks spawn<CR>", desc = "toggletasks: spawn" },
            ["<Leader>tsc"] = { "<Cmd>Telescope toggletasks select<CR>", desc = "toggletasks: select" },
            ["<Leader>tse"] = { "<Cmd>Telescope toggletasks edit<CR>", desc = "toggletasks: edit config" },
          },
        },
      },
    },
```

map keys with AstroCore

**Review by mehalter — CHANGES_REQUESTED** [review]

**Review by mehalter — APPROVED** [review]

**Comment by vbmaarten** [comment]
Suggestions have been applied, and the result has been tested locally

**Review by mehalter — CHANGES_REQUESTED** [review]
Last thing, thanks so much! I forget, I don't think `init` implies lazy = true in lazy.nvim

**Comment by vbmaarten** [comment]
> Last thing, thanks so much! I forget, I don't think `init` implies lazy = true in lazy.nvim

Applied and tested 👍

**Review by mehalter — APPROVED** [review]

**Pull request merged**